awx/ui_next/src/components/ResourceAccessList/ResourceAccessListItem.jsx

```diff
@@ -24,19 +24,13 @@ const DataListItemCells = styled(PFDataListItemCells)`
   align-items: start;
 `;
 
-class ResourceAccessListItem extends React.Component {
-  static propTypes = {
+function ResourceAccessListItem({ accessRecord, onRoleDelete, i18n }) {
+  ResourceAccessListItem.propTypes = {
```

Comment:
It feels a little funny to modify this from within the function, though I suppose it works. I typically drop this down below the end of the component